io.catenax.asset_tracker_links/1.0.0/AssetTrackerLinks.ttl

```diff
@@ -0,0 +1,86 @@
+#######################################################################
+# Copyright (c) 2022 BASF SE
+# Copyright (c) 2022 Henkel AG & Co. KGaA
+# Copyright (c) 2022 ZF Friedrichshafen AG
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+#######################################################################
+
+
+@prefix bamm: <urn:bamm:io.openmanufacturing:meta-model:2.0.0#>.
+@prefix bamm-c: <urn:bamm:io.openmanufacturing:characteristic:2.0.0#>.
+@prefix bamm-e: <urn:bamm:io.openmanufacturing:entity:2.0.0#>.
+@prefix unit: <urn:bamm:io.openmanufacturing:unit:2.0.0#>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix : <urn:bamm:io.catenax.asset_tracker_links:1.0.0#>.
+
+:AssetTrackerLinks a bamm:Aspect;
+    bamm:preferredName "Asset Tracker Links"@en;
+    bamm:description "Represent the link between Asset and IoT Device (tracker) with timestamps of pairing and unpairing."@en;
+    bamm:properties (:catenaXId :childParts);
+    bamm:operations ();
+    bamm:events ().
+:catenaXId a bamm:Property;
```

Comment:
be prepared that this might be migrated to a shared aspect in the future, as a consolidation of the catenax id modeling is currently taking place. Thus, it might be required to introduce a new model version the n the shared aspect is available.

Reply:
noted. Thanks!